src/Parser/Rogue/Assassination/CONFIG.js

```diff
@@ -16,9 +16,13 @@ export default {
   description: (
     <React.Fragment>
       <Warning>
-        Hey there! A good basis has been implemented for this spec, but it needs to be fleshed out more to provide all the feedback possible.<br /><br />
-
-        This spec needs a focused maintainer. If you want to give it a try, check <a href="https://github.com/WoWAnalyzer/WoWAnalyzer">GitHub</a> for more information.
+        Assassination rogue analysis isn't complete yet. Analysis should pick up most general mistakes, however:
+        <ul>
+          <li>there is no in-depth analysis for the Pre-Patch </li>
+          <li>target values may be tuned incorrectly for things like energy waste or downtime. </li>
+        </ul>
+        <br />
+        If something is missing, incorrect, or inaccurate, please report it on <a href="https://github.com/WoWAnalyzer/WoWAnalyzer/issues/new">GitHub</a> or contact <kbd>@Cloake</kbd> on <a href="https://discord.gg/AxphPxU">Discord</a>.<br /><br />
```

Comment:
is there a reason for having two `<br />` tags at the end of this message?

Reply:
Edited. Here's a screenshot:

![image](https://user-images.githubusercontent.com/29325796/43347659-2af36848-91ab-11e8-9425-8aa70632ddf7.png)

Here's a sample log URL:
https://www.warcraftlogs.com/reports/YDpAw13ZtQ4yhrPF#fight=9&type=damage-done

Reply:
As for the DPS gained: it is a talent, and a difficult one to calculate actual (as opposed to estimated, from average DPS) DPS gain from. I can implement the functionality in a later PR, but don't believe it's necessary for analysis of the spec at the top level of play.